beta/package.json

```diff
@@ -22,9 +22,9 @@
     "@docsearch/css": "3.0.0-alpha.41",
     "@docsearch/react": "3.0.0-alpha.41",
     "@headlessui/react": "^1.3.0",
-    "@mdx-js/mdx": "^1.6.16",
-    "@mdx-js/react": "^1.6.16",
-    "@mdx-js/tag": "^0.20.3",
+    "@mdx-js/mdx": "^2.0.0-rc.1",
+    "@mdx-js/react": "^2.0.0-rc.1",
+    "@mdx-js/tag": "^1.0.0-alpha.6",
```

Comment:
this is no longer a thing, it was removed in v1! `@mdx-js/react` was the replacement